widget/embedded/src/components/WalletStatefulConnect/ExperimentalChain.tsx

```diff
@@ -0,0 +1,36 @@
+import { i18n } from '@lingui/core';
+import { Button, Divider, MessageBox } from '@rango-dev/ui';
+import React from 'react';
+
+interface PropTypes {
+  displayName?: string;
+  onConfirm: () => void;
+}
+
+export function ExperimentalChain(props: PropTypes) {
```

Comment:
I think name of this component does not specify what it is. Obviously this component is not an experimental chain.

Reply:
It was `ModalExpermintalChain`, I only removed the `Modal`. That would be great to give me your suggestion as well.